install/bootstrap_livecd_remote_iso.md

```diff
@@ -472,18 +505,13 @@ Some files are needed for generating the configuration payload. See the [Command
           --cabinets-yaml cabinets.yaml \
           --hmn-mtn-cidr 10.104.0.0/17 \
           --nmn-mtn-cidr 10.100.0.0/17 \
```

Comment:
I think there is at least one more option that is required that I don't see there.   We should test these out again with a 1.2 CSI.

Reply:
Yeah, I don't know the last time someone has run through those options. I didn't make any changes to it in this PR, so I'd suggest we tackle that with a different ticket.

Reply:
Agreed, this can be handled separately.   Just wanted to bring it up as something we should get fixed.